dlgr/griduniverse/templates/grid.html

```diff
@@ -23,6 +23,8 @@
     settings.walls = "{{ experiment.grid.wall_type }}";
     settings.show_grid = {% if experiment.grid.show_grid %}true{% else %}false{% endif %};
     settings.visibility = {{ experiment.grid.visibility }};
+    settings.chat_visibility_threshold = {{ experiment.grid.chat_visibility_threshold }};
+    settings.spatial_chat = {{ experiment.grid.spatial_chat }};
```

Comment:
@vlall I think the `True is not defined` error you're seeing is because this line should be:

```
settings.spatial_chat = {% if experiment.grid.spatial_chat %}true{% else %}false{% endif %};
```

It'd be nice if there were a more general way of converting Python truth values to JS truth values via Flask... maybe there is.

Reply:
Cool thanks, added a commit